src/plugins/GUITestBase/src/GTUtilsWorkflowDesigner.h

```diff
@@ -101,25 +98,25 @@ class GTUtilsWorkflowDesigner {
     static QStringList getPaletteGroupEntriesNames(HI::GUITestOpStatus &os, const QString &groupName);
 
     //add to scene
-    static void addSample(HI::GUITestOpStatus &os, const QString &sampName, QWidget const *const parentWidget = NULL);
-    static void addAlgorithm(HI::GUITestOpStatus &os, QString algName, bool exactMatch = false, bool useDragAndDrop = false);
+    static void addSample(HI::GUITestOpStatus &os, const QString &sampName, QWidget *parentWidget = nullptr);
+    static void addAlgorithm(HI::GUITestOpStatus &os, const QString &algName, bool exactMatch = false, bool useDragAndDrop = false);
     static WorkflowProcessItem *addElement(HI::GUITestOpStatus &os, const QString &algName, bool exactMatch = false);
     static WorkflowProcessItem *addElementByUsingNameFilter(HI::GUITestOpStatus &os, const QString &elementName, bool exactMatch = false);
 
     //returns center of worker on workflow scene in global coordinates
-    static QPoint getItemCenter(HI::GUITestOpStatus &os, QString itemName);
+    static QPoint getItemCenter(HI::GUITestOpStatus &os, const QString& itemName);
```

Comment:
Sometimes you write `const QString&`, sometimes `const QString &`

Reply:
I always write String& but the UGENE code style requires String &. Sometimes I forgot to run a formatter. Fixed